src/main/java/nl/tudelft/pixelperfect/event/parameter/EventParameterCollection.java

```diff
@@ -75,6 +76,24 @@ public String toString() {
   }
 
   /**
+   * Generate a debug string representation for the parameters.
+   * 
+   * @return the debug string.
+   */
+  public String toDebugString() {
+    StringBuilder sb = new StringBuilder();
+    if (map.isEmpty()) {
+      sb.append("None");
+    } else {
+      for (EventParameter param : map.values()) {
+        sb.append(param.getNumberValue());
+        sb.append(",");
```

Comment:
Line 90 gives a PMD warning: Avoid appending characters as strings in StringBuffer.append.